### DISCUSSION TITLE
The Colorful Chameleon

### DISCUSSION THREAD

**Discussion by meviru** [meviru/badge-hunting#12] [Galaxy Trivia] [answered]
Imagine you have a paragraph that wants to show off its new blue outfit. How can you change all `<p>` elements to wear blue, using just one line of JavaScript?

**Comment by shynekomaid** [comment, marked as answer]
```
document.querySelectorAll('p').forEach(p => p.style.color = 'blue');
```
Details:

1. document.querySelectorAll('p'): This method selects all <p> elements in the document. It returns a NodeList, which is a collection of elements.

2..forEach(p => ... ): The forEach method is used to iterate over each element in the NodeList. It takes a callback function as an argument, which is executed for each element.

3 p => p.style.color = 'blue': This is an arrow function that takes a single parameter p (representing each <p> element). Inside the function, p.style.color = 'blue' sets the color property of the element's style to 'blue'.

So, this one-liner code effectively selects all <p> elements and changes their text color to blue by iterating through the NodeList and updating each element's style.
